macros/SpecName.ejs

```diff
@@ -1123,6 +1123,10 @@ var specList = {
         name : 'WebVR',
         url  : 'https://mozvr.com/webvr-spec/'
```

Comment:
Hrm, the 1.0 version actually doesn't seem to exist anymore. So I'll delete that. But I'll keep the other one as 1.1, as 2.0 is also on the (distant) horizon